### PR TITLE
Updates per Issues 23910, 23918, and 23919

### PR DESCRIPTION
**Pull request by ahardin-rh**
Addresses:

- https://github.com/openshift/openshift-docs/issues/23919
- https://github.com/openshift/openshift-docs/issues/23918
- https://github.com/openshift/openshift-docs/issues/23910

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@jtaleric @mffiedler PTAL

**Review by mffiedler — COMMENTED** [review]
LGTM.  Really helps clarify this tricky area.

File: modules/recommended-configurable-storage-technology.adoc [hunk 1]
```diff
@@ -60,21 +60,22 @@ storage would be an anti-pattern. One volume per elasticsearch is required.
 
 [NOTE]
 ====
-A scaled registry is an {product-title} registry where three or more pod replicas are running.
+A scaled registry is an {product-title} registry where two or more pod replicas are running.
```
Comment (codyhoag) [inline modules/recommended-configurable-storage-technology.adoc:63]: s/pod/Pod

I know this isn't part of the direct changes, but figured I'd flag it. :slightly_smiling_face:

**Review by codyhoag — COMMENTED** [review]
One non-related nit: all changes LGTM!

**Pull request merged**